cola-maven-plugin-test/src/test/java/com/github/bmsantos/maven/cola/ColaInjectedTest.java

```diff
@@ -0,0 +1,62 @@
+package com.github.bmsantos.maven.cola;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Named;
+
+import com.github.bmsantos.core.cola.story.annotations.ColaInjected;
+import com.github.bmsantos.core.cola.story.annotations.Given;
+import com.github.bmsantos.core.cola.story.annotations.Then;
+import com.github.bmsantos.core.cola.story.annotations.When;
+import com.github.bmsantos.core.cola.story.processor.StoryProcessor;
+import com.google.inject.Inject;
+import org.junit.Before;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertThat;
+
+@ColaInjected
+public class ColaInjectedTest extends BaseColaTest {
+
```

Comment:
This test is confusing.  The feature describes both adding and multiplying, but I don't understand why.  And the Integer/String conversion and using a for loop to get the result is very terse. Also, all you do with injectedString is print it.